tst/testinstall/package.tst

```diff
@@ -516,6 +516,23 @@ new methods:
   mockpkg_Property( ... )*
 
 
+# Cite() expects GAPInfo.Date to be of the form "YYYY-MM-DD" or "YYYY-Mon-DD" (or "today")
+gap> IsDateFormatValid := function( datestring )
+>      local months, val;
+>      if datestring = "today" then
+>        return true;
+>      fi;
+>      val:= SplitString( datestring, "-" );
+>      if Length( val ) <> 3 then
+>        return false;
+>      fi;
+>      months:= [ "Jan", "Feb", "Mar", "Apr", "May", "Jun",
```

Comment:
There is a global variable `NameMonth` that could be used here (and I guess in `BibEntry`). Not important, though